test/e2e/common_test.go

```diff
@@ -320,7 +320,7 @@ func (p *PodmanTestIntegration) createArtifact(image string) {
 	}
 	dest := strings.Split(image, "/")
 	destName := fmt.Sprintf("/tmp/%s.tar", strings.Replace(strings.Join(strings.Split(dest[len(dest)-1], "/"), ""), ":", "-", -1))
-	fmt.Printf("Caching %s at %s...", image, destName)
+	fmt.Printf("Caching %s at %s...\n", image, destName)
```

Comment:
This is immediately followed by a verbose `podman pull`, which spits out `Running: etc` on the same line. My logformatter does not see this command and does not perform its helpful highlights.